ote/src/cljs/ote/views/passenger_transportation.cljs

```diff
@@ -45,7 +45,13 @@
     :layout :row}
 
    {:name ::t-service/name
-    :type :string}))
+    :type :string}
+
+   {:style {:width "350px"}
```

Comment:
hard coded pixel width should be moved to a style ns

Reply:
Fixed